streaming/java/pom.xml

```diff
@@ -0,0 +1,273 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.ray</groupId>
+  <artifactId>ray-streaming</artifactId>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
Should we redefine the version from now on？

Reply:
We should follow ray/java pom version now, but it'd better modified and bump to released version.